util/chplenv/utils.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Without it, this file trigger the following lintian error: https://lintian.debian.org/tags/executable-not-elf-or-script.html